### ISSUE TITLE
Reorder `setPageAttributes()` parameters

### ISSUE THREAD

**Issue by weshooper** [fusionspim/php-demandware-xml#61]
From:

`public function setPageAttributes(?string $pageTitle = null, ?string $pageDescription = null, ?string $pageKeywords = null, ?string $pageUrl = null): void`

To:

`public function setPageAttributes(?string $pageUrl = null, ?string $pageTitle = null, ?string $pageDescription = null, ?string $pageKeywords = null): void`

This would be a breaking change, however pages tend to always have a URL, but may not need keywords. Therefore it'd mean not needing to pass `null` for keywords.

**Comment by ziadoz** [comment]
I agree we should re-order the parameters. We also never send keywords, as they're kind of defunct as a form of meta data for SEO, so we'd be able to simplify the code usage in Fusions.
